lib/data/sender.js

```diff
@@ -98,7 +98,11 @@ class DataTrackSender extends DataTrackTransceiver {
       }
     });
     this._clones.forEach(clone => {
-      clone.send(data);
+      try {
+        clone.send(data);
+      } catch (error) {
```

Comment:
curious- why was this catch necessary? in what cases do we expect errors? Maybe worth adding a console.log here?

Reply:
@makarandp0 In integration tests, sometime when you disconnect from a Room, if the timing is right, trying to send data on RTCDataChannels whose RTCPeerConnection is closed throws an error. This makes sure that the tests don't fail because of it. Regarding logging, we should do it once we thread the logging module down to this level.